recipes/fontconfig/all/conanfile.py

```diff
@@ -13,8 +13,16 @@ class FontconfigConan(ConanFile):
     homepage = "https://gitlab.freedesktop.org/fontconfig/fontconfig"
     topics = ("conan", "fontconfig", "fonts", "freedesktop")
     settings = "os", "compiler", "build_type", "arch"
-    options = {"shared": [True, False], "fPIC": [True, False]}
-    default_options = {"shared": False, "fPIC": True}
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "with_nls": [True, False],
```

Comment:
nls = native language support
Is this used for some executable or in the library?
if it's only for a tool/executable, then I would simply disable nls unconditionally (and don't use libgettext).

Reply:
Hi @madebr, thank you for taking the time to review my change.
I'll honestly say I didn't know whether it's actually used or not. I looked around for use of this feature elsewhere and found that bison, flex, yasm and, ironically, gettext, libgettext, libidn, libidn2 all disable i18n features.
gdbm has a with_nls option and I cargo culted that as it seemed like the least invasive option.
I took a look at the fontconfig code and the [POTFILES.in](https://github.com/freedesktop/fontconfig/blob/master/po/POTFILES.in) file makes me think that i18n is only used for a few executables and very few strings in the library. There is only one translation in chinese.
I'm heartbroken as I think i18n is a requirement for software in this century but the lack of enough languages in the library itself make me think it's never been much of a requirement to have it translated. So I'll follow your suggestion.